DependencyInjection/FOSRestExtension.php

```diff
@@ -88,7 +90,15 @@ private function loadForm(array $config, XmlFileLoader $loader, ContainerBuilder
     {
         if (!empty($config['disable_csrf_role'])) {
             $loader->load('forms.xml');
-            $container->getDefinition('fos_rest.form.extension.csrf_disable')->replaceArgument(1, $config['disable_csrf_role']);
+
+            $definition = $container->getDefinition('fos_rest.form.extension.csrf_disable');
+            $definition->replaceArgument(1, $config['disable_csrf_role']);
+
+            if (method_exists(AbstractType::class, 'getBlockPrefix')) {
```

Comment:
What about adding a comment such as `BC` to remind us to remove it next time we bump sf version?